nohup: ignoring input
/home/jsammet/UKB_brain_mri_raw/mri_env/lib/python3.6/site-packages/torch/nn/modules/loss.py:520: UserWarning: Using a target size (torch.Size([20])) that is different to the input size (torch.Size([20, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
/home/jsammet/UKB_brain_mri_raw/mri_env/lib/python3.6/site-packages/torch/nn/modules/loss.py:520: UserWarning: Using a target size (torch.Size([14])) that is different to the input size (torch.Size([14, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
/home/jsammet/UKB_brain_mri_raw/mri_env/lib/python3.6/site-packages/torch/nn/modules/loss.py:520: UserWarning: Using a target size (torch.Size([19])) that is different to the input size (torch.Size([19, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
/home/jsammet/UKB_brain_mri_raw/mri_env/lib/python3.6/site-packages/torch/nn/modules/loss.py:520: UserWarning: Using a target size (torch.Size([3])) that is different to the input size (torch.Size([3, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.mse_loss(input, target, reduction=self.reduction)
Traceback (most recent call last):
  File "main.py", line 71, in <module>
    test_saliency(model, dataset, test_indices, params, criterion)
  File "/home/jsammet/UKB_Brain_Iron/src/training.py", line 182, in test_saliency
    saliency, _ = torch.max(image.grad.data.abs(), dim=1)
AttributeError: 'NoneType' object has no attribute 'data'
